hooks/boilerplate.py

```diff
@@ -25,18 +25,20 @@
 def PrintError(*err):
   print(*err, file=sys.stderr)
 
-def file_passes(filename, extention, ref, regex):
+def file_passes(filename, extention, ref, regexs):
     try:
         f = open(filename, 'r')
     except:
         return False
 
-    data = f.readlines()
+    data = f.read()
 
     # remove build tags from the top of Go file
     if extention == "go":
```

Comment:
(also python seriously lets you use this variable declared in another scope? that is totally not cool.)